File: Prak 7/C/PRAK605-2210817110005-MIZZUDDINABDISSALAM.c
#include <stdio.h>

int main(){
    int ordo, a, b, c, isi, hasil;
    scanf("%d", &ordo);
    int matrik1[ordo][ordo], matrik2[ordo][ordo];
    printf("\nMatriks A\n");
    for(a = 0; a < ordo; a++){
        for(b = 0; b < ordo; b++){
            scanf("%d", &isi);
            matrik1[a][b] = isi;}}
    printf("\nMatriks B\n");
    for(a = 0; a < ordo; a++){
        for(b = 0; b < ordo; b++){
            scanf("%d", &isi);
            matrik2[a][b] = isi;}}
    printf("\nMatriks A x B\n");
    for(a = 0; a < ordo; a++){
        for(b = 0; b < ordo; b++){
            for(c = 0, hasil = 0; c < ordo; c++){
                hasil += matrik1[a][c] * matrik2[c][b];}
            printf("%d ", hasil);}
    printf("\n");}
return 0;}

File: Prak 7/Python/PRAK605-2210817110005-MIZZUDDINABDISSALAM.py
ordo = int(input())
matrik1 = []
print("Matriks A")
for i in range(0, ordo):
    isi1 = list(map(int, input().split()))
    matrik1.append(isi1[0: ordo])
matrik2 = []
print("Matriks B")
for i in range(0, ordo):
    isi2 = list(map(int, input().split()))
    matrik2.append(isi2[0: ordo])
print("\nMatriks A x B")
for i in range(0, ordo):
    for j in range(0, ordo):
        hasil = 0
        for k in range(0, ordo):
            hasil += matrik1[i][k] * matrik2[k][j]
        print(hasil, end=' ')
    print()
